utils/logutils.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func LogExit(msg string, fmts ...interface{}) {
+	log.Fatalf(msg, fmts...)
```

Comment:
Ah ok; I didn’t realize you sent a patch upstream. Let’s hold off on this patch until that gets merged, then.

-Erik

> On May 26, 2015, at 5:06 PM, Madhav Puri <notifications@github.com> wrote:
> 
> In utils/logutils.go <https://github.com/contiv/netplugin/pull/79#discussion_r31092564>:
> 
> > @@ -0,0 +1,12 @@
> > +package utils
> > +
> > +import (
> > +	"os"
> > +
> > +	log "github.com/Sirupsen/logrus"
> > +)
> > +
> > +func LogExit(msg string, fmts ...interface{}) {
> > +	log.Fatalf(msg, fmts...)
> @erikh <https://github.com/erikh>, I am still wondering if we need this utility function. Unless user set's log level to be PanicLevel, logrus will exit.
> 
> also just fyi I created a PR Sirupsen/logrus#188 <https://github.com/Sirupsen/logrus/pull/188> in logrus to fix the above behavior, once it is merged we can update our godeps to get the fix.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/79/files#r31092564>.
>